### DISCUSSION TITLE
Hello Githubers

### DISCUSSION THREAD

**Discussion by Johnalvarez713** [community/community#43358] [New to GitHub] [answered]
Hello all! As you can probably tell by the badge right next to my profile picture that I am new to github. I am new to coding and programing as well. I am just a person who loves to learn. So quick question. What do I actually do with the zip files that I download? I have tried extracting one but it doesnt seem to grasp what I am suppose to do with these files? I unpacked the Zip and 28 folders popped out. I dont see an exe and then the read me is a completely different file type. It .md If someone could please give me some advice I really want to grasp and enjoy what Github has to offer. This is my 2nd time on the platform. The first time I downloaded something, didnt know what to do with it and was too emberassed to ask. As a matter of fact I dont even think we can message each other. I am just finding out about this community that we can post in. Thanks a bunch. I did look in the search bar for "How to install" "How to download" "Tutorials" I think we could really use a getting started thread.

**Comment by Ankush11903** [comment, marked as answer]
Hello @Johnalvarez713.
If you are facing difficulty in zip file then. U can just push it into your git by cloning that repo.
Or can pull that repo.


You can check this for reference
https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository



If it helps you please mark this discussion as answered.
Thanks.

**Reply by queenofcorgis** [reply]
Thanks for jumping in so quickly @Ankush11903  and sharing a helpful doc 😄

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community @Johnalvarez713 🎉 

The Community is here to help and glad to see you've already got an answer. Thanks for marking the answer - hope to see you around with any more questions you may have.
